src/main/resources/data/ancientology/recipes/bronze_ingot_from_block.json

```diff
@@ -0,0 +1,19 @@
+{
+	"type": "minecraft:crafting_shaped",
+	
+	"pattern":
+	[
+		"I"
+	],
+	
+	"key":
+	{
+		"I": {"item": "ancientology:bronze_block"}
+	},
+	
+	"result": 
+	{ 
+		"count": 9,
+		"item": "ancientology:bronze_ingot"
+	}
+}
```

Comment:
You already made a datagen'd recipe for this?

Reply:
There is one to make a block from ingots... Don't see one for a block to ingots